python/BioSimSpace/Sandpit/Exscientia/Protocol/_dummy.py

```diff
@@ -0,0 +1,17 @@
+__all__ = ["Dummy"]
+from ._protocol import Protocol as _Protocol
+
+
+class Dummy(_Protocol):
```

Comment:
Since the special behaviour for the protocol seems to come from the `Process` objects anyway, do you even need `Dummy`? You could just say `if protocol is not None` instead of `if not isinstance(protocol, Dummy)` etc.

Reply:
My thinking is that later one we might need different dummy protocol for 

- Generate the topology and coordinate
- Not generate anything
- Or more different cases such as only generate the reference coordinate

So I keep it as dummy protocol so we could add more things later if there is need.

Another reason is that there are numerous checks assume Protocol is of Protocol class so having a dummy protocol would mean no need to change all those thing.

Reply:
Okay that makes sense. It feels like there should be a better design pattern but I can't think of anything so I guess it's fine to merge.